tests/utils/test_polar.py

```diff
@@ -0,0 +1,45 @@
+from os.path import abspath, dirname, join
+
+import gemmi
+import pandas as pd
+import pytest
+
+import reciprocalspaceship as rs
+
+
+def sgtbx_polar_classification():
+    """
+    Helper function for generating Hermann-Mauguin (xhm) symbols with
+    corresponding polar classifications
+    """
+    data = ["..", "data", "sgtbx", "sgtbx_polar.csv"]
+    inFN = abspath(join(dirname(__file__), *data))
+    ref = pd.read_csv(inFN)
+    return ref.iterrows()
+
+
+@pytest.fixture(params=sgtbx_polar_classification())
+def polar_by_xhm(request):
```

Comment:
admittedly, i do not understand fixtures all that well, but i found this file surprisingly difficult to read. i think this might be helped by adding 
```
Parameters
---------------
request : ??????
```
to the docstring.